script.js

```diff
@@ -0,0 +1,40 @@
+function calculate() {
+    
+    value1 = parseInt(document.getElementById("value1").value);
+    value2 = parseInt(document.getElementById("value2").value);
+  
+    document.getElementById("answer").innerHTML = operate(value1, value2);
+
+}
+function operate(value1,value2)
+{
+    var operator = document.getElementById("operator").value;
+    if (operator == '+')
+     {
+        return value1 + value2;
+    } 
+    else if (operator == '-') 
+    {
+        return value1 - value2;
+    } else if (operator == '*') 
+    {
+        return value1 * value2;
+    } else if (operator == '/')
+     {
+        return value1 / value2;
+    }
```

Comment:
Use a switch statement here for cleaner code. Also use === instead of == but that isn't something we'd grade down for right now.